compiler/src/parsing/parser.messages

```diff
@@ -231,7 +231,7 @@ program: MODULE UIDENT EOL FOREIGN WASM LIDENT COLON LIDENT ARROW
 ## FOREIGN WASM id_str COLON typ
 ##
 
-Expected the keyword `from` followed by a file path.
+Expected the keyword `from` followed by an external module name surrounded by `"`.
```

Comment:
This is related to the foreign wasm statement? I don't like that we are using "external module name" maybe "external wasm namespace"?